server/graphql/v2/mutation/OrderMutations.js

```diff
@@ -571,6 +602,19 @@ const orderMutations = {
               },
             },
           );
+
+          // Update paymentMethodId in transactions for Added Funds
+          if (addedFundOrders.length > 0 && isCollective) {
```

Comment:
Why only for type=COLLECTIVE?

Reply:
If it's a `USER` we will have the `opencollective/host` payment method type. We shouldn't change that. Only for collectives I thought we should change the payment method type (to `opencollective/collective` type).

Reply:
What if it's a fund, an organization, an event, a project?